homeassistant/components/tankerkoenig/__init__.py

```diff
@@ -17,11 +17,7 @@ async def async_setup_entry(
     """Set a tankerkoenig configuration entry up."""
     hass.data.setdefault(DOMAIN, {})
 
-    coordinator = TankerkoenigDataUpdateCoordinator(
-        hass,
-        name=entry.unique_id or DOMAIN,
-        update_interval=DEFAULT_SCAN_INTERVAL,
-    )
+    coordinator = TankerkoenigDataUpdateCoordinator(hass, entry, DEFAULT_SCAN_INTERVAL)
```

Comment:
Why remove the parameter names?

Reply:
because these are positional arguments, those not needed to address by name